src/CreeDictionary/DatabaseManager/cree_inflection_generator.py

```diff
@@ -32,10 +29,7 @@ def expand_inflections(
         lemma, word_class = lemma_and_wordclass_from_analysis(analysis)
 
         if word_class.has_inflections():
-            paradigm_name = convert_crkeng_word_class_to_paradigm_name(word_class)
-            generated_analyses = list(
-                paradigm_manager.all_analyses(paradigm_name, lemma)
-            )
+            raise Exception("convert_crkeng_word_class_to_paradigm_name is gone")
```

Comment:
Why even have this line? Is the `WordClass` enum even used or instantiated anymore?

Reply:
This is some dead code that I am leaving for cleanup in a future PR.

Reply:
This line came about because merging the main branch into this one pulled in a call to `convert_crkeng_word_class_to_paradigm_name()` which was already gone from this branch.